operator/pkg/ciliumendpointslice/controller.go

```diff
@@ -63,15 +69,27 @@ type Controller struct {
 	// CES requests going to api-server, ensures a single CES will not be proccessed
 	// multiple times concurrently, and if CES is added multiple times before it
 	// can be processed, this will only be processed only once.
-	queue     workqueue.RateLimitingInterface
-	rateLimit rateLimitConfig
+	fastQueue     workqueue.RateLimitingInterface
+	standardQueue workqueue.RateLimitingInterface
+
+	rateLimit   rateLimitConfig
+	rateLimiter workqueue.RateLimiter
 
 	enqueuedAt     map[CESName]time.Time
 	enqueuedAtLock lock.Mutex
 
 	wp *workerpool.WorkerPool
 
 	metrics *Metrics
+
+	defaultCESSyncTime time.Duration
+
+	priorityNamespaces map[string]struct{}
+
+	cond     sync.Cond
+	condLock lock.Mutex
```

Comment:
We could potentially remove this condLock and initilize in-line with cond? see diff below
```suggestion
```